tapestry.js

```diff
@@ -30,11 +31,11 @@ const // calculated
 var dataset, root, svg, links, nodes,               // Basics
     path, pieGenerator, arcGenerator,               // Donut
     linkForce, collideForce, force,                 // Force
-    tapestrySlug, saveProgress = true,      // Cookie
+    tapestrySlug, saveProgress = true, saveProgressCounter = 0,     // Cookie
```

Comment:
Can you change the `// Cookie` comment to something more explanatory like ` // Saving` or something?